app/services/ipam.rb

```diff
@@ -0,0 +1,16 @@
+module IPAM
+  MODES = {:dhcp => N_('DHCP'), :db => N_('Internal DB'), :eui64 => N_('EUI-64'), :none => N_('None')}
```

Comment:
Scratch that, actually. There are a few other things that would need to be changed in order for this to work (like auto-detecting the IP type based on how it looks), so let's do that in the next PR concerning UI. I'll just test it manually.